### ISSUE TITLE
Python packages empty non-empty list

### ISSUE THREAD

**Issue by vsfeedback** [microsoft/PTVS#8065]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Python-packages-empty-non-empty-list-/10785713)._

---
In the side bar of Python Environments there is, after selecting Packages (PyPI), the list of installed packages.  
When VS loads the python project the list is correctly shown.  
After installing one additional package, all the entries of the list simply disappear, while it’s still possible to install new packages (correct output is shown after installing new packages).

I found that the problem was solved only after adding a new Python Environment. So, this sort of “bug” was shown only with one single python environment in the list. Sometime the package list still shows to be empty, but after double clicking on the desired python environment it is correctly re-loaded. 

---
#### Feedback Bot on 11/10/2024, 06:12 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
#### Francesco Fambri solved on 11/8/2024, 01:15 AM, undefined votes: 

<p>report here again the workaround:</p>
<p>I found that the problem was solved only after adding a new Python Environment. So, this sort of “bug” was shown only with one single python environment in the list. Sometime the package list still shows to be empty, but after double clicking on the desired python environment it is correctly re-loaded.</p>

**Comment by StellaHuang95** [comment]
Closing as duplicate of https://github.com/microsoft/PTVS/issues/7715
